src/classes/com/sun/tdk/jcov/Merger.java

```diff
@@ -992,7 +993,8 @@ private int handleEnv_(EnvHandler opts) throws EnvHandlingException {
         outTestList = null;
         if (opts.isSet(DSC_OUTPUT_TEST_LIST)) {
             outTestList = opts.getValue(DSC_OUTPUT_TEST_LIST);
-            Utils.checkFileNotNull(outTestList, "output testlist filename", Utils.CheckOptions.FILE_CANWRITE, Utils.CheckOptions.FILE_NOTISDIR, Utils.CheckOptions.FILE_PARENTEXISTS);
+            Utils.checkFileNotNull(outTestList, "output testlist filename",
```

Comment:
If this is an _output_ test list (like I assume for merging with test scales, does it have to exist)?

Reply:
I now see that you have changed the semantics of FILE_CANWRITE. This is probably OK then.